doc/source/api/visualization/index.rst

```diff
@@ -3,13 +3,13 @@
 Visualization
 =============
 
-Post-processing of Fluent results can be done with either Fluent in-built
-post-processing capabilities or with the PyVista/MatplotLib integration.
+Postprocessing of Fluent results can be done with either built-in Fluent
+postprocessing capabilities or with a PyVista or Matplotlib integration.
```

Comment:
```suggestion
postprocessing capabilities, PyVista, or Matplotlib integration.
```
?

Reply:
@dnwillia-work Isn't the "integration" only for PyVista or Matplotlib given that the other option is built-in Fluent postprocessing capabilities. Perhaps I'm misunderstanding what's intended.

Reply:
If I understood correctly, you can use either of the three backends for post-processing results. Commiting suggestion and merging :)